src/Exceptionless.Core/Extensions/OrganizationExtensions.cs

```diff
@@ -77,6 +77,7 @@ public static class OrganizationExtensions {
         }
 
         public static async Task<bool> IsOverRequestLimitAsync(string organizationId, ICacheClient cacheClient, int apiThrottleLimit) {
+            if (apiThrottleLimit < 0 || apiThrottleLimit == int.MaxValue) return false;
```

Comment:
We shouldn't be checking for -1 just Int32.MaxValue; We need to normalize -1 in the settings.